packages/open-payments/src/config.ts

```diff
@@ -0,0 +1,5 @@
+export default {
+  OPEN_PAYMENTS_OPEN_API_URL:
```

Comment:
want to make sure the same URL is used for type generation and (following) open API response validation